runtime/parser2/declaration.go

```diff
@@ -121,14 +126,18 @@ func parseAccess(p *parser) ast.Access {
 	case keywordPub:
 		// Skip the `pub` keyword
 		p.next()
-		p.skipSpaceAndComments(true)
+		p.parseTrivia(triviaOptions{
```

Comment:
When I look at this change, it makes sense. Because I have the context, I know you wanted to replace with `parseTrivia` and what it does it to skip space and comments. 

Since it doesn't have the `parseDocStrings: true` option, it's actually doing `skipSpaceAndComments`. The problem with the function name `parseTrivia` is that if I have never seen this PR, I would question what is `Trivia`, why parsing it?

I would suggest to refactor the `skipSpaceAndComments` and use `parseTrivia` to implement `skipSpaceAndComments` if possible, so that we don't need to introduce this change here.

Reply:
I renamed the function, because it was quite verbose, especially when I added the options struct, e.g. code looked like `skipSpaceAndComments(skipSpaceAndCommentsOptions{...`. 

Trivia is the best name I found, as it's also the term used in the Swift AST. I can revert the renaming.

How do you propose to refactor the function?